source/Uncontrolled.tsx

```diff
@@ -25,4 +25,4 @@ const Uncontrolled: FC<UncontrolledProps> = (props: UncontrolledProps) => {
 // there is no need to zoom images.
 export default typeof window !== 'undefined'
   ? memo(Uncontrolled)
-  : (props: UncontrolledProps) => props.children
+  : (props: UncontrolledProps) => <Fragment>props.children</Fragment>
```

Comment:
Good call! Did you know about `<></>`?

```suggestion
  : (props: UncontrolledProps) => <>props.children</>
```

Reply:
Good morning, yep I know about `<></>` I know it's a shorthand for Fragment, but nothing more than that, I'll update it 👍🏼